nixos-up.py

```diff
@@ -243,7 +243,7 @@ def refresh_block_index():
 # it avoids putting hashed passwords into the world-readable nix store. See
 # https://discourse.nixos.org/t/introducing-nixos-up-a-dead-simple-installer-for-nixos/12350/11?u=samuela *)
 hashed_password = subprocess.run(["mkpasswd", "--method=sha-512", password], check=True, capture_output=True, text=True).stdout.strip()
-password_file_path = f"/mnt/etc/passwordFile-{username}"
+password_file_path = f"/mnt/etc/hashedPasswordFile-{username}"
 with open(password_file_path, "w") as f:
   f.write(hashed_password)
 os.chmod(password_file_path, 600)
```

Comment:
Let's try fixing this such that it can't happen again in the future. I'm thinking:

```suggestion
password_file_path = f"/etc/hashedPasswordFile-{username}"
with open(f"/mnt{password_file_path}", "w") as f:
  f.write(hashed_password)
os.chmod(f"/mnt{password_file_path}", 600)
```
and then
```
f" hashedPasswordFile = \"{password_file_path}\";",
```
in line 257. Or something in that vein.

Reply:
Done. Please review.